samples/boards/ti/cc1352r_sensortag/src/main.c

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2020 Friedt Professional Engineering Services, Inc
+ *
```

Comment:
Might be good to add a little comment about what the sample actually does.

Reply:
Will do

Reply:
Done!